sdclientapi/__init__.py

```diff
@@ -55,7 +55,8 @@ def json_query(proxy_vm_name: str, data: str, timeout: Optional[int] = None) ->
         stdout=PIPE,
         stderr=PIPE,
     )
-    p.stdin.write(data.encode("utf-8"))
+    if p.stdin is not None:
+        p.stdin.write(data.encode("utf-8"))
```

Comment:
I also considered the following solution:

```
    # We tell mypy to ignore the following line because it reports a false positive:
    #    > error: Item "None" of "Optional[IO[bytes]]" has no attribute "write"
    # We know that stdin is a writeable stream object and that it is not None because
    # we set the stdin argument to "PIPE" (see https://docs.python.org/3/library/subprocess.html)
    p.stdin.write(data.encode("utf-8"))  # type: ignore
```

But prefer to keep mypy code line skips to a minimum.